src/types/gecko-profile.js

```diff
@@ -177,41 +177,34 @@ export type GeckoCounter = {|
   |},
 |};
 
-export type GeckoProfileMeta = {|
-  interval: Milliseconds,
+/* This meta object is used in subprocesses profiles.
+ * Using https://searchfox.org/mozilla-central/rev/7556a400affa9eb99e522d2d17c40689fa23a729/tools/profiler/core/platform.cpp#1829
+ * as source of truth. (Please update the link whenever there's a new property).
+ * */
+export type GeckoProfileShortMeta = {|
+  version: number,
   startTime: Milliseconds,
   shutdownTime: Milliseconds | null,
-  abi: string,
-  // The extensions property landed in Firefox 60, and is only optional because
-  // older profile versions may not have it. No upgrader was written for this change.
-  extensions?: GeckoExtensionMeta,
   categories: CategoryList,
```

Comment:
Aside: we should probably get rid of this property on subprocesses.

Reply:
And `version` as well, this is super useless to have it here...